docs/nddata/utils.rst

```diff
@@ -182,16 +178,18 @@ non-overlapping arrays will raise a
 positions in the cutout array that do not overlap with the ``data``
 array will be filled with ``fill_value``.  In ``'trim'`` mode only the
 overlapping elements are returned, thus the resulting cutout array may
-be smaller than the requested ``shape``.
+be smaller than the requested ``size``.
 
 The default uses ``mode='trim'``, which can result in cutout arrays
-that are smaller than the requested ``shape``::
+that are smaller than the requested ``size``::
 
     >>> data2 = np.arange(20.).reshape(5, 4)
     >>> c1 = Cutout2D(data2, (0, 0), (3, 3), mode='trim')
```

Comment:
Very minor comment: should `c1` be `cutout1` to be consistent with the rest of the doc/docstring changes? Obviously not critical..